lib/find_retryables.ts

```diff
@@ -120,8 +152,16 @@ const main = async () => {
   await checkRetryablesOneOff()
 }
 
-// Calling main
-main()
+// Parsing command line arguments
+const options = yargs(process.argv.slice(2))
+  .options({
+    fromBlock: { type: 'number', default: 0 },
+    toBlock: { type: 'number', default: 0 },
```

Comment:
Default for "toBlock" could be the latest block (you can calculate it in "main" if toBlock == 0)